README/CREDITS

```diff
@@ -455,6 +455,11 @@ N: Burt Holzman
 E: burt@fnal.gov
 D: Contribution to TWebFile,rpdutils and sockets.
 
+N: Raphael Isemann
+E: risemann@cern.ch
```

Comment:
Isn't this email account going to become obsolete soon? Maybe it makes sense to add a more durable email. Up to @Teemperor...

Reply:
Good catch, teemperor@gmail.com is probably better suited.